src/Microsoft.AspNetCore.Testing/xunit/MinOsVersionSkipConditionAttribute.cs

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Microsoft.Win32;
+
+namespace Microsoft.AspNetCore.Testing.xunit
+{
+    /// <summary>
+    /// Skips a test if the OS is the given type (Windows) and the OS version is less than specified.
+    /// E.g. Specifying Window 10.0 skips on Win 8, but not on Linux. Combine with OSSkipConditionAttribute as needed.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Class | AttributeTargets.Assembly, AllowMultiple = true)]
+    public class MinOsVersionSkipConditionAttribute : Attribute, ITestCondition
```

Comment:
I don't feel a need for `SkipCondition` to be in the name. It makes me think this is the minimum version on which to **skip** the test, rather than the minimum version on which the test **can** run. Maybe something like `MinimumOSVersionAttribute` would be fine? I like the way that makes the test look:

```csharp
[ConditionalFact]
[MinimumOSVersion(OperatingSystems.Windows, "10.0.12345.0")]
public void Test()
{
}
```